employees/common/strings.py

```diff
@@ -111,12 +109,13 @@ class ReportValidationStrings(NotCallableMixin, Enum):
     WORK_HOURS_SUM_FOR_GIVEN_DATE_FOR_SINGLE_AUTHOR_EXCEEDED = _(
         "Sum of work hours of all reports from a given day for single author must not exceed 24."
     )
-    WORK_HOURS_MIN_VALUE_NOT_EXCEEDED = _(
-        f"Minimum value for work hours for single report must not be less than {int(ReportModelConstants.MIN_WORK_HOURS.value.total_seconds() / 60)} minutes"
-    )
+    WORK_HOURS_MIN_VALUE_NOT_EXCEEDED = _("Ensure this value is greater than or equal to 0:15:00.")
```

Comment:
As far as I remember we should cut off last numbers (seconds in this case) and just show HH:MM